models/model_functions.R

```diff
@@ -14,6 +14,23 @@ get_moon_data <- function(){
   return(moons)
 }
 
+get_future_moons <- function(moons){
+  #Get dates of future new moons from navy website
+  year = tail(moons$year,1)
```

Comment:
change these to `most_recent_year` and `most_recent_month` to be more descriptive